docs/welcome/why-kubeedge.md

```diff
@@ -37,7 +37,7 @@ KubeEdge is composed of these components:
 - **[MetaManager](./architecture/edge/metamanager):** the message processor between edged and edgehub. It is also responsible for storing/retrieving metadata to/from a lightweight database (SQLite).
 - **[ServiceBus](./architecture/edge/servicebus)**: a HTTP client to interact with HTTP servers (REST), offering HTTP client capabilities to components of cloud to reach HTTP servers running at edge.
 - **[DeviceController](./architecture/cloud/device_controller)**: an extended kubernetes controller which manages devices so that the device metadata/status data can be synced between edge and cloud.
-
+- **MQTT Broker**: A server that handles MQTT messages for efficient communication with IoT devices.
```

Comment:
MQTT Broker is not a component of KubeEdge, it's not suitable to put it here.

Reply:
@Shelley-BaoYue This component is resolved now. Thank you for your feedback.

Reply:
@Shelley-BaoYue @anvithks  Would be thankful if you could review the PR and give any suggestion if needed.